src/components/Account/AccountProfilePage.tsx

```diff
@@ -177,7 +178,8 @@ class AccountProfilePage extends React.Component<IProps, null> {
 
                 return errors;
               }}
-              onSubmit={this.handleSubmit.bind(this)}
+              onSubmit={this.handleFormSubmit()}
```

Comment:
This still creates a new function object on each render, and so I'm not sure what is the improvement here.

The pattern to use here is to ` bind` the function int he constructor, as we do for example here: https://github.com/daostack/alchemy/blob/dev/src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx#L97

(So you would undo the change you made in `handleFormSubmit`, remove the ` bind` in onSubmit, and instead bind int he constructor)

Reply:
@tibetsprague You once advised me to use the method I'm using.  Can you please confirm @jellegerbrandy's suggestion, to make sure we're all on the same page?  Thanks!

(Just a note BTW, documenting best practices for fixing lint errors like this one is one of the documentation tasks on your plate, Tibet  :-)  )

Reply:
@jellegerbrandy   @tibetsprague  My understanding of the problem here is not that the function is being called on every render, but worse, the fact the react thinks a new function is created actually causes any subcomponents depending on it (to which the new function is given) to be rendered more often.

Reading some more:  There is a similar pattern with the creation of new instances of _anything_, like variables created in the render method, and that are then passed into subcomponents.

Reply:
well, yes, indeed.